src/patient-registration/patient-registration.resource.tsx

```diff
@@ -12,6 +12,17 @@ export function savePatient(abortController: AbortController, patient: Patient)
   });
 }
 
+export function saveRelationships(abortController: AbortController, relationships: Object) {
```

Comment:
Is there a better typing for `relationships: Object`? See `Patient` for more details on specifying a custom type.

Reply:
Unfortunately here we are passing the tempRelationships and not relationship object....

Reply:
What does `tempRelationships `look like?

Reply:
[Done](https://github.com/openmrs/openmrs-esm-patient-registration/pull/20/files#diff-1f098aba5021d89ce5c54d2038db8f80R23)